### ISSUE TITLE
How to apply `update_interval` in ApexOptimWrapper or AmpOptimWrapper

### ISSUE THREAD

**Issue by BinhuiXie** [open-mmlab/mmaction2#2409]
### What is the problem this feature will solve?

I want to keep the batch size unchanged while my GPU memory is limited. So, could I accumulate the gradient for some steps and then backpropagate? How?

### What is the feature?

Hi, thank you for the beautiful project.

**Comment by hukkai** [comment]
For gradient accumulation, you can refer to https://github.com/open-mmlab/mmaction2/blob/main/docs/en/advanced_guides/customize_optimizer.md#gradient-accumulation

**Comment by BinhuiXie** [comment]
thx~
